_rest_routes.inc.php

```diff
@@ -1128,6 +1128,32 @@
         RestConfig::apiLog($return);
         return $return;
     },
+    "GET /fhir/Provenance/:uuid" => function ($uuid, HttpRestRequest $request) {
+        if ($request->isPatientRequest()) {
+            // only allow access to data of binded patient
+            $return = (new FhirProvenanceRestController())->getOne($uuid, $request->getPatientUUIDString());
+        } else {
+            // TODO: it seems like regular users should be able to grab authorship / provenance information
+            RestConfig::authorization_check("admin", "super");
+            $return = (new FhirProvenanceRestController())->getOne($uuid);
+        }
+        RestConfig::apiLog($return);
+        return $return;
+    },
+    // NOTE: this GET request only supports requests with an _id parameter.  FHIR inferno test tool requires the 'search'
+    // property to support which is why this endpoint exists.
+    "GET /fhir/Provenance" => function ($uuid, HttpRestRequest $request) {
+        if ($request->isPatientRequest()) {
+            // only allow access to data of binded patient
+            $return = (new FhirProvenanceRestController())->getAll($request->getQueryParams(), $request->getPatientUUIDString());
+        } else {
+            // TODO: it seems like regular users should be able to grab authorship / provenance information
+            RestConfig::authorization_check("admin", "super");
+            $return = (new FhirProvenanceRestController())->getOne($request->getQueryParams());
```

Comment:
is this supposed to be getAll?

Reply:
@adunsulag ,
oh goodness, not sure where my brain went tonight (I brought this PR in even though i noted this issue :) ). If this needs fixing, let me know and I can bring it in or just bring in a PR and will bring it in.

Reply:
@bradymiller I run into a major setback with Provenance on the Bulk-FHIR so I'm going to have to do some work here again anyways.  I'll do a PR and we can fix this.

Reply:
@adunsulag I'd like to talk to you about Provenance!